src/services/LIFI/Lifi.ts

```diff
@@ -0,0 +1,42 @@
+import { DepositAction, Step, WithdrawAction } from '@lifinance/types'
+import axios from 'axios'
+
+import { isDeposit, isWithdraw } from './typeguards'
+
+class LIFI {
+  findRoutes = async (deposit: DepositAction, withdraw: WithdrawAction): Promise<Step[][]> => {
+    if (!isDeposit(deposit)) {
+      throw Error('Invalid Deposit Type')
+    }
+    if (!isWithdraw(withdraw)) {
+      throw Error('Invalid Withdraw Type')
+    }
+    const result = await axios.post<any>(process.env.REACT_APP_API_URL + 'transfer', {
+      deposit,
+      withdraw,
+    })
+    return result.data as Step[][]
+  }
```

Comment:
We don't need to use `<any>` here if we know what result we expect. You can define `Step[][]` instead.